types/rdfjs__parser-jsonld/index.d.ts

```diff
@@ -4,7 +4,7 @@
 //                 tpluscode <https://github.com/tpluscode>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 
-import { Context } from 'jsonld/jsonld-spec';
+import { Context } from 'jsonld/jsonld-spec.js';
```

Comment:
I suppose this is fine, but it occurs to me that the type definitions for `jsonld` do not match the actual shape of the package, since the `jsonld` package has no file called `jsonld-spec.js`. Its existence is purely type-space, so it seems strange to request an import of a `.js` file that doesn't actually exist.

Reply:
Interesting observation. Sounds link it reveals a deficiency of the module resolution algorithm?

Reply:
Would you rather have me fix the jsonld package?

Reply:
ping @rbuckton

Reply:
No, that's not necessary.